content/download/intro-designers.md

```diff
@@ -1,6 +1,8 @@
 ---
 layout: download/intro
 sidebar: intro-designers-sidebar.md
+paddingTop: true
+borderBottom: true
```

Comment:
We will need to do this a different way. We do not add css with props.

Reply:
fixed